.github/workflows/build_and_deploy.yml

```diff
@@ -21,65 +19,69 @@ jobs:
-        context: .
-
-  deploy_review :
-        name: Deploy to review environment
-        runs-on: ubuntu-latest
-        if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy') }}
-        needs: [build]
-        environment:
-          name: review
-          url: ${{ steps.get-app-url.outputs.url1 }}
-
-        steps:
-          - name: Checkout code
-            uses: actions/checkout@v4
-
-          - uses: ./.github/actions/deploy-environment
-            id: deploy
-            with:
-              environment: review
-              docker-image: ${{ needs.build.outputs.docker-image-tag }}
-              azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
-              pull-request-number: ${{ github.event.number }}
-
-          - uses: azure/login@v1
-            with:
-              creds: ${{ secrets.AZURE_CREDENTIALS }}
-
-          - name: Seed Review App
-            shell: bash
-            if: github.event.number != ''
-            run: |
-                az aks get-credentials -g s189t01-tsc-ts-rg -n s189t01-tsc-test-aks
-                kubectl exec -n bat-qa deployment/itt-mentor-services-${{ github.event.number }} -- sh -c "export GIAS_CSV_BASE_URL=https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public  && export PUBLISH_BASE_URL=https://qa.api.publish-teacher-training-courses.service.gov.uk// && cd /app && /usr/local/bin/rake db:seed"
-
-          - name: Get application URL
-            id: get-app-url
-            shell: bash
-            run: |
-              url1="https://track-and-pay-${{ github.event.number }}.test.teacherservices.cloud"
-              echo "url1=${url1}" >> $GITHUB_OUTPUT
-              url2="https://manage-school-placements-${{ github.event.number }}.test.teacherservices.cloud"
-              echo "url2=${url2}" >> $GITHUB_OUTPUT
-
-          - name: Post comment to Pull Request ${{ github.event.number }}
-            if: ${{ github.event_name == 'pull_request' }}
-            uses: marocchino/sticky-pull-request-comment@v2
-            with:
-              header: aks
-              message: |
-                      Review app  track and pay deployed to <https://track-and-pay-${{ github.event.number }}.test.teacherservices.cloud>
-                      Review app  school placements deployed to <https://manage-school-placements-${{ github.event.number }}.test.teacherservices.cloud>
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build and push docker image
+        id: build-image
+        uses: DFE-Digital/github-actions/build-docker-image@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          context: .
+
+  deploy_review:
+    name: Deploy to review environment
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy') }}
+    needs: [build]
+    environment:
+      name: review
+      url: ${{ steps.get-app-url.outputs.url1 }}
```

Comment:
As discussed, this unique URL is confusing. Shall we remove it? from here and the action output

Reply:
Updated with https://github.com/DFE-Digital/itt-mentor-services/pull/172/commits/54a4c2e8fd0ff9cc11ccf1390e9622371c0686b9 :D